### PR TITLE
Update cli commands

### LINKED ISSUE #15144: The md style for some cli command dose not be treated well.
**This is a Bug Report**

**Problem:**
The md style for some cli command dose not be treated well.
https://kubernetes.io/zh/docs/tasks/inject-data-application/define-command-argument-container/

**Proposed Solution:**
Reformat the code style for cli commands.

**Page to Update:**
https://kubernetes.io/zh/docs/tasks/inject-data-application/define-command-argument-container/

### PR DESCRIPTION
**Pull request by Potapy4**
Fixes #15144

### CODE REVIEW

File: content/zh/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 1]
```diff
@@ -34,14 +34,12 @@ content_template: templates/task
 {{< code file="commands.yaml" >}}
 
 1. 基于YAML文件创建一个Pod：
-    
+
     ```shell
-    kubectl create -f https://k8s.io/docs/tasks/inject-data-application/commands.yaml
+    kubectl apply -f https://k8s.io/examples/pods/commands.yaml
```
Comment (zhangqx2010) [inline content/zh/docs/tasks/inject-data-application/define-command-argument-container.md]: Maybe just change `create` to `apply` and keep the original file url?

File: content/zh/docs/tasks/inject-data-application/define-command-argument-container.md [hunk 2]
```diff
@@ -59,7 +57,7 @@ content_template: templates/task
```
Comment (zhangqx2010) [inline content/zh/docs/tasks/inject-data-application/define-command-argument-container.md]: This part still need be modified. Looks not good in netlify preview:

  获取正在运行的 pod

    ```shell
    kubectl get pods
    ```

 查询结果显示在command-demo这个Pod下运行的容器已经启动完成

**Comment by zhangqx2010** [comment]
Seems still not correctly showed on page...

website:
image
vs
md file preview:
image

**Comment by Potapy4** [comment]
> Seems still not correctly showed on page...
> 
> website:
> image
> vs
> md file preview:
> image

should be fixed now

**Pull request merged**